plugins/modules/aci_vrf_leak_internal_subnet.py

```diff
@@ -0,0 +1,385 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
```

Comment:
Add your details at line 4  -> # Copyright: (c) 2023, <<NAME>> (@ciscousername) <email>
before the line # GNU General Public License

Reply:
replace line 4 by -> # Copyright: (c) 2021, Anvitha Jain (@anvitha-jain) <anvjain@cisco.com>

replace my name -> by your name